demisto_sdk/commands/download/downloader.py

```diff
@@ -160,271 +141,316 @@ def __init__(
         **kwargs,
     ):
         self.output_pack_path = output
-        self.input_files = [input] if isinstance(input, str) else input
+        self.input_files = input if isinstance(input, list) else [input]
         self.regex = regex
         self.force = force
-        self.download_system_item = system
+        self.download_system_items = system
         self.system_item_type = item_type
         self.insecure = insecure
         self.list_files = list_files
-        self.all_custom_content = all_custom_content
+        self.download_all_custom_content = all_custom_content
         self.run_format = run_format
         self.client = None
-        self.custom_content_temp_dir = mkdtemp()
-        self.system_content_temp_dir = mkdtemp()
-        self.all_custom_content_objects: List[dict] = list()
-        self.files_not_downloaded: List[list] = list()
-        self.custom_content: List[dict] = list()
-        self.pack_content: Dict[str, list] = {
-            entity: list() for entity in CONTENT_ENTITIES_DIRS
-        }
-        self.num_merged_files = 0
-        self.num_added_files = 0
         self.init = init
         self.keep_empty_folders = keep_empty_folders
         if is_sdk_defined_working_offline() and self.run_format:
             self.run_format = False
             logger.info(
-                "format is not supported when the DEMISTO_SDK_OFFLINE_ENV environment variable is set, skipping it."
+                "Formatting is not supported when the DEMISTO_SDK_OFFLINE_ENV environment variable is set, Skipping..."
             )
 
     def download(self) -> int:
         """
-        Downloads custom content data from Demisto to the output pack in content repository.
-        :return: The exit code
+        Downloads custom content data from XSOAR to the output path provided.
+
+        Returns:
+            int: Exit code. 1 if failed, 0 if succeeded
         """
-        exit_code: int = self.download_manager()
-        self.remove_traces()
-        return exit_code
+        return self.download_manager()
```

Comment:
if all it does is call `download_manager()`, why do we need it?

Reply:
We probably don't, I'm just not sure if we have a convention for command structures or something?
Should I just move the content of `download_manager` to `download`?

Or call `return downloader.download_manager()` instead of `return downloader.download()` from `__main__.py`?